src/benchmarklib/benchmark_table_encoder.cpp

```diff
@@ -130,13 +132,30 @@ bool BenchmarkTableEncoder::encode(const std::string& table_name, const std::sha
   auto encoding_performed = false;
```

Comment:
not sure, probably better to make this atomic?

Reply:
I don't think it needs to be atomic. It's a one-sided switch, that only gets turned on -- or not. Multiple threads writing it does simultaneously doesn't change anything, since they can only set it to true anyway. And when the value is returned, all threads are done already.

Reply:
yepp

Reply:
Ok, you win. While this can't really break, tsan is unhappy about it, too. I rather change it than add a suppression.